frontend/src/app/features/invite-user-modal/invite-user-modal.module.ts

```diff
@@ -24,6 +24,9 @@ import { SuccessComponent } from './success/success.component';
 export function initializeServices(injector:Injector) {
   return function () {
     const inviteUserAugmentService = injector.get(OpInviteUserModalAugmentService);
+    document.addEventListener('turbo:load', () => {
+      inviteUserAugmentService.setupListener();
+    });
```

Comment:
We already have a global listener on `turbo:load` where we reinitialize angular services. Please add your call [there](https://github.com/opf/openproject/blob/release/16.2/frontend/src/app/app.module.ts#L221), so that we have it bundled together and also works after a `turbo:render`.

Reply:
Thanks, I though that there should be some place like that